modules/ROOT/pages/queries/case.adoc

```diff
@@ -0,0 +1,250 @@
+[[case]]
+= Conditional expressions (CASE)
+:description: This page describes how to use the CASE expression with Cypher.
+
+Generic conditional expressions can be expressed in Cypher using the `CASE` construct.
+Two variants of `CASE` exist within Cypher: the simple form, which allows an expression to be compared against multiple values, and the generic form, which allows multiple conditional statements to be expressed.
+
+[NOTE]
+====
+`CASE` can only be used as part of xref:clauses/return.adoc[] or xref:clauses/with.adoc[] if you want to use the result in a subsequent clause.
+====
+
+[[case-example]]
+== Example graph
+
+The following graph is used for the examples below:
+
+image:case_graph.svg[width="500",role="middle"]
+
+To recreate the graph, run the following query against an empty Neo4j database:
+
+[source, cypher, role=test-setup]
+----
+CREATE
+  (alice:Person {name:'Alice', age: 38, eyes: 'brown'}),
+  (bob:Person {name: 'Bob', age: 25, eyes: 'blue'}),
+  (charlie:Person {name: 'Charlie', age: 53, eyes: 'green'}),
+  (daniel:Person {name: 'Daniel', eyes: 'brown'}),
+  (eskil:Person {name: 'Eskil', age: 41, eyes: 'blue'}),
+  (alice)-[:KNOWS]->(bob),
+  (alice)-[:KNOWS]->(charlie),
+  (bob)-[:KNOWS]->(daniel),
+  (charlie)-[:KNOWS]->(daniel),
+  (bob)-[:MARRIED]->(eskil)
+----
+
+[[case-simple]]
+== Simple `CASE`
+
+The simple `CASE` form is used to compare an expression against multiple values.
+The expressions are evaluated by the `WHEN` operator until a `MATCH` is found.
+If no match is found, the expression in the `ELSE` operator is returned.
+If there is no `ELSE` case and no match is found, `null` will be returned.
+
+[[case-simple-syntax]]
+=== Syntax 
+
+[source, syntax]
+----
+CASE test
+  WHEN value THEN result
+  [WHEN ...]
+  [ELSE default]
+END
+----
+
+*Arguments:*
+[options="header"]
+|===
+| Name | Description
+
+| `test`
+| A valid expression.
+
+| `value`
+| An expression whose result will be compared to `test`.
+
+| `result`
+| This is the expression returned as output if `value` matches `test`.
```

Comment:
```suggestion
| The expression to be returned as output if `value` matches `test`.
```

Reply:
Or just "The expression returned as output..."